tests/settings/SectionTest.py

```diff
@@ -138,11 +140,25 @@ def test_append_to_sections(self):
         self.assertEqual(len(sections), 1)
         self.assertEqual(len(sections['default'].contents), 1)
 
-        append_to_sections(sections, 'test1', 'val', 'origin', 'default')
+        append_to_sections(sections,
+                           'test1',
+                           'val',
+                           'origin',
+                           section_name='default')
         self.assertIn('default', sections)
         self.assertEqual(len(sections), 1)
         self.assertEqual(len(sections['default'].contents), 2)
 
+        append_to_sections(sections,
+                           'test1',
+                           'val2',
+                           'origin',
+                           to_append=True,
+                           section_name='section')
+        self.assertEqual(str(sections['section']['test1']), 'val2')
+        sections['section'].set_default_section(sections)
+        self.assertEqual(str(sections['section']['test1']), 'val,val2')
```

Comment:
and especially when adding tests we shouldn't mix them into existing testcases but make new ones, this will only lead to more crowded and less understandable test cases I fear